kanidmd/score/src/lib.rs

```diff
@@ -686,6 +687,13 @@ pub async fn create_server_core(config: Configuration, config_test: bool) -> Res
     if config_test {
         admin_info!("this config rocks! 🪨 ");
     } else {
+        let enable_metrics = match &config.metrics_listener {
+            Some(_) => {
```

Comment:
I think you could do "is_some()" instead.

Reply:
Oh neat, have done this! 👍🏻